mergify_engine/actions/merge_base.py

```diff
@@ -338,7 +344,7 @@ async def _run(
                 return check_api.Result(
                     check_api.Conclusion.CANCELLED,
                     "The pull request has been removed from the queue",
-                    "The queue conditions cannot be satisfied due to failing checks. "
+                    "The queue conditions cannot be satisfied due to failing checks or checks timeout. "
```

Comment:
can we know which one?

Reply:
The details is on the other check-run, this message continues with https://github.com/Mergifyio/mergify-engine/blob/main/mergify_engine/actions/queue.py#L48.